doc/introduction/setup-dev.md

```diff
@@ -0,0 +1,28 @@
+# Setup a dev environment
+
+## Using docker
+
+The docker-compose is already ok for dev.
+
+So using:
+
+```bash
+docker-compose build
```

Comment:
```suggestion
DOCKER_BUILDKIT=1 docker-compose build
```

Reply:
hum if we need DOCKER_BUILDKIT, I will add it to the .env

Reply:
oh no, we cannot, ok let's document then.